cciskel-duffy

```diff
@@ -21,7 +22,7 @@ import argparse
 
 url_base="http://admin.ci.centos.org:8080"
 api_key=open(os.path.expanduser('~/duffy.key')).read().strip()
-cached_state_path = os.path.expanduser('~/duffy-state-cache.json')
+cached_state_path = os.path.expanduser('~/.duffy-state-cache.json')
```

Comment:
Not sure about this...it's not really a config file.

If we're going to change this I'd say we create a `~/jenkins-state` directory or something where we also put the API key file etc.